components/pipeline-service/production/stone-prd-m01/deploy.yaml

```diff
@@ -1634,7 +1774,7 @@ spec:
             )" \
             --dry-run=client \
             -o yaml | kubectl apply -f -
-        image: quay.io/konflux-ci/appstudio-utils:ab6b0b8e40e440158e7288c73aff1cf83a2cc8a9@sha256:24179f0efd06c65d16868c2d7eb82573cce8e43533de6cea14fec3b7446e0b14
+        image: quay.io/redhat-appstudio/appstudio-utils:dbbdd82734232e6289e8fbae5b4c858481a7c057
```

Comment:
is this intentional ?? .... I think konflux-ci is the new approved way ... maybe this is getting picked up from the pipeline service repo and should be reverted there ?

Reply:
Good catch. I compared that with staging and considered it good, but it isn't. The problem is #3848 updated the deploy.yaml files directly, instead of the refs in the pipeline-service repo and now our update is overriding it. I'll create the fix in the service repo, but we can't consume it without the OSP bump (or at least without revering it in both service and infra repos and staging clusters, which I'm not willing to do). So I'll create the change in the service repo and revert those specific changes in the deploy.yaml files. @rcerven FYI.